### PR TITLE
add description on methods

### PR DESCRIPTION
**Pull request by agallou**
cf #2 

add description on methods.

done with this script : https://gist.github.com/agallou/e8d74efde698703d9db5
it adds/replaces the help inside the `-- help --` and `-- endhelp ---`.

It takes the description from the asserters.txt file generated from the documentation.

It that's ok for you i'll do the same for the properties.

//cc @mnapoli @jubianchi

### CODE REVIEW

**Comment by mnapoli** [comment]
:+1: awesome!

**Comment by jubianchi** [comment]
@agallou https://gist.github.com/agallou/e8d74efde698703d9db5#gistcomment-1664299

**Comment by jubianchi** [comment]
To me, there is at least two things to do:
- [ ] Commit the script in this repository
- [ ] The script should automatically generate its `asserters.txt` file from the doc (via a git clone or archive download)

**Comment by agallou** [comment]
@jubianchi script commited with our fix to avoid the `-- help --`. All `-- help --` removed. The script now clones the documentation repository.

File: bin/updateDoc [hunk 1]
```diff
@@ -0,0 +1,317 @@
...
+			1,
+			function() use ($helps, $refectionClass){
+				return getClassHelp($refectionClass, $helps);
+			}
+		);
+		replaceInFile($file, $refectionClass->getDocComment(), $newClassDocComment, $refectionClass->getShortName());
+	}
+}
+
+passthru('rm -rf tmp && mkdir -p tmp && cd tmp && git clone git@github.com:atoum/atoum-documentation.git && cd atoum-documentation && ./builddoc text');
```
Comment (jubianchi) [inline bin/updateDoc]: can you use the `https` clone url as it will be clonable by anyone without any restriction.

**Comment by agallou** [comment]
@jubianchi the clone is now done via https

**Pull request merged**